src/utils/fulfill.ts

```diff
@@ -829,3 +846,7 @@ export const getAdvancedOrderNumeratorDenominator = (
 
   return { numerator, denominator };
 };
+
+export const getTagFromDomain = (domain: string) => {
+  return keccak256(toUtf8Bytes(domain)).slice(0, 10).replace(/^(0x)/, "");
```

Comment:
Does this regex look right?

Reply:
you don't really need the regex, just do `.slice(2, 10)`